### PR TITLE
added section on scalajs-stubs

### PR DESCRIPTION
**Pull request by ozeebee**
see https://github.com/scala-js/scala-js-website/issues/178

### CODE REVIEW

File: doc/project/cross-build.md [hunk 1]
```diff
@@ -90,3 +90,15 @@ lazy val foo = crossProject.in(file(".")).
...
   )
 {% endhighlight %}
+
+## Exporting shared classes to JavaScript
+
+When working with shared classes, you may want to export some of them to JavaScript.
+This is done with annotations as explained in [Export Scala.js APIs to JavaScript](../interoperability/export-to-javascript.html).
+
+These annotations are part of the Scala.JS library which is not available on the Scala JVM project.
+In order for the annotated classes to compile on the JVM project, you should add the scalajs-stubs library to your JVM dependencies as "provided" (used only during compilation and not included at runtime):
```
Comment (gzm0) [inline doc/project/cross-build.md]: I think it is very important here to stress the point that a library will **not** depend on the `scalajs-stubs` library. I think you can stress this more.
Also, you might want to mention, that the annotations are not static, i.e. they will not appear in the `.class` files at all.

Comment (sjrd) [inline doc/project/cross-build.md]: Except it does. At least it survives in POM files and gets resolved further. But it's not put on the run-time classpaths.

To completely remove the dependency, one needs an additional POM postprocessor (there's an sbt setting for that).

Comment (gzm0) [inline doc/project/cross-build.md]: Ouch... I guess I learned something :)

File: doc/project/cross-build.md [hunk 2]
```diff
@@ -90,3 +90,15 @@ lazy val foo = crossProject.in(file(".")).
     libraryDependencies += "com.lihaoyi" %%% "scalatags" % "0.4.3"
   )
 {% endhighlight %}
+
+## Exporting shared classes to JavaScript
+
+When working with shared classes, you may want to export some of them to JavaScript.
+This is done with annotations as explained in [Export Scala.js APIs to JavaScript](../interoperability/export-to-javascript.html).
+
+These annotations are part of the Scala.JS library which is not available on the Scala JVM project.
```
Comment (sjrd) [inline doc/project/cross-build.md]: The proper spelling is `Scala.js` with lowercase `js`.

**Comment by sjrd** [comment]
That's all.

**Comment by sjrd** [comment]
OK could please squash the commits in one single commit?

**Comment by ozeebee** [comment]
Done !

**Comment by sjrd** [comment]
LGTM

**Pull request merged**